hlink/linking/model_exploration/link_step_train_test_models.py

```diff
@@ -429,7 +830,7 @@ def _save_otd_data(
             print("There were no true negatives recorded.")
 
     def _create_otd_data(self, id_a: str, id_b: str) -> dict[str, Any] | None:
```

Comment:
This is a good opportunity to rename the "OTD data" to something easier to understand. Maybe "suspicious data" would be clearer?

Reply:
Yes, I replaced "otd" with "suspicious".